experiments/README.md

```diff
@@ -0,0 +1,41 @@
+Experiment scripts are only compatible with Linux.
+
+### Setup
+
+Phase 1 requires [AWS CLI](https://aws.amazon.com/cli/) because it downloads data from AWS S3.
+
+### Phase 1: Download RL (PPO2) expert policies and AIRL/GAIL reward models.
+
+Use `experiments/download_models.sh`. (Downloads to `data/{expert,reward}_models/`).
+Expert policies are used in Phase 2 to generate demonstrations.
+Reward models are used in Phase 4 for transfer learning.
+
+Want to use other policies or reward models for Phase 2 or 4? 
+  * New policies can be trained using `experiments/train_experts.sh`. Enter the optional command suggested at the end of the script to upload new files to S3. Or, manually patch `data/expert_models` using the script's output files.
+  * New reward models are generated in Phase 3 `experiments/imit_benchmark.sh`. Enter the optional command suggested at the end of the script to upload new files to S3. Or, manually overwrite `data/reward_models` using the script's output files.
+Either follow the command printed by the script to update S3 or manually overwrite `data/{expert,reward}_models/`.
```

Comment:
Do you need this line and the explanation in the previous bullet points? Inclined to delete the last two sentences in the bullet point and expand on it in this line.

Reply:
To elaborate, I'm thinking something like:

```
  * New policies can be trained using `experiments/train_experts.sh`. 
  * New reward models are generated in Phase 3 `experiments/imit_benchmark.sh`.
+ For both scripts, you can enter the optional command suggested at the end of the script to upload new files to S3. Or, you can manually patch `data/{expert,reward}_models` using the script's output files.
```

Reply:
Thanks